src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/manualdetermination/ClaimantLipManualDeterminationFormGenerator.java

```diff
@@ -0,0 +1,79 @@
+package uk.gov.hmcts.reform.civil.service.docmosis.manualdetermination;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.civil.documentmanagement.DocumentManagementService;
+import uk.gov.hmcts.reform.civil.documentmanagement.model.CaseDocument;
+import uk.gov.hmcts.reform.civil.documentmanagement.model.DocumentType;
+import uk.gov.hmcts.reform.civil.documentmanagement.model.PDF;
+import uk.gov.hmcts.reform.civil.enums.PaymentFrequencyClaimantResponseLRspec;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.model.PaymentBySetDate;
+import uk.gov.hmcts.reform.civil.model.docmosis.DocmosisDocument;
+import uk.gov.hmcts.reform.civil.model.docmosis.manualdetermination.ClaimantLipManualDeterminationForm;
+import uk.gov.hmcts.reform.civil.service.docmosis.DocmosisTemplates;
+import uk.gov.hmcts.reform.civil.service.docmosis.DocumentGeneratorService;
+import uk.gov.hmcts.reform.civil.service.docmosis.TemplateDataGenerator;
+import uk.gov.hmcts.reform.civil.service.docmosis.utils.ClaimantResponseUtils;
+
+import java.time.LocalDate;
+
+import static java.util.Objects.nonNull;
+import static uk.gov.hmcts.reform.civil.service.docmosis.DocmosisTemplates.CLAIMANT_LIP_MANUAL_DETERMINATION_PDF;
+
+@Service
+@Getter
+@RequiredArgsConstructor
+public class ClaimantLipManualDeterminationFormGenerator implements TemplateDataGenerator<ClaimantLipManualDeterminationForm> {
+
+    private final DocumentManagementService documentManagementService;
+    private final DocumentGeneratorService documentGeneratorService;
+
+    public CaseDocument generate(CaseData caseData, String authorisation) {
+        ClaimantLipManualDeterminationForm templateData = getTemplateData(caseData);
+
```

Comment:
no need for these empty lines. Context of the code for document and templates is the same.

Reply:
fixed